report-schema-v100.json

```
@@ -0,0 +1,600 @@
+                                        "type": "array",
+                                        "title": "The external links schema",
+                                        "description": "List of external links mentioned in the report entry.",
+                                        "default": [],
+                                        "examples": [
+                                            [
+                                                {
+                                                    "url": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/package_manifest/codereadylinuxbuilder-repository.",
+                                                    "title": "CodeReady Linux Builder repository"
+                                                }
+                                            ]
+                                        ],
+                                        "additionalItems": true,
+                                        "items": {
+                                            "$id": "#/properties/entries/items/anyOf/0/properties/detail/properties/external/items",
+                                            "anyOf": [
+                                                {
+                                                    "$id": "#/properties/entries/items/anyOf/0/properties/detail/properties/external/items/anyOf/0",
+                                                    "type": "object",
+                                                    "title": "The first external link",
+                                                    "description": "An external link.",
+                                                    "default": {},
+                                                    "examples": [
+                                                        {
+                                                            "url": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/package_manifest/codereadylinuxbuilder-repository.",
+                                                            "title": "CodeReady Linux Builder repository"
+                                                        }
+                                                    ],
+                                                    "required": [
+                                                        "url",
+                                                        "title"
+                                                    ],
+                                                    "properties": {
+                                                        "url": {
+                                                            "$id": "#/properties/entries/items/anyOf/0/properties/detail/properties/external/items/anyOf/0/properties/url",
+                                                            "type": "string",
```

What about to add the `uri` format to make validation even more strict?"

```suggestion
                                                            "type": "string",
                                                            "format": "uri",
```

ref: https://json-schema.org/understanding-json-schema/reference/string.html#resource-identifiers